biome.json

```diff
@@ -0,0 +1,38 @@
+{
+  "$schema": "https://biomejs.dev/schemas/1.4.1/schema.json",
+  "files": {
+    "ignore": [
+      "**/dist",
+      "**/node_modules",
+      "**/fixtures",
+      "**/.vscode-test",
+      ".github",
+      ".changeset",
+      ".idea/"
+    ],
+    "ignoreUnknown": true
+  },
+  "formatter": {
+    "lineWidth": 100
+  },
+  "javascript": {
+    "formatter": {
+      "quoteStyle": "single",
+      "trailingComma": "es5"
+    }
+  },
+  "json": {
+    "formatter": {
+      "indentStyle": "space"
+    },
+    "parser": {
+      "allowComments": true
+    }
```

Comment:
This allows us to understand and format `.vscode/extensions.json` (it has comments)